net.sourceforge.jpdftweak.jPdfTweak.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id" : "net.sourceforge.jpdftweak.jPdfTweak",
+  "branch" : "stable",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "19.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "command": "net.sourceforge.jpdftweak.jPdfTweak",
+  "finish-args" : [
+    "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+	"--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--filesystem=host",
+    "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+  ],
+  "modules" : [
+    {
+      "name" : "openjdk",
+      "buildsystem" : "simple",
+      "build-commands" : [ "/usr/lib/sdk/openjdk11/install.sh" ]
+    },
+    {
+      "name": "jpdftweak",
+      "buildsystem": "simple",
+      "build-commands": [
+        "unzip jpdftweak-dist.zip -d dist",
+        "mkdir -p /app/jpdftweak",
+        "cp -r dist/* /app/jpdftweak",
+        "for size in 16 32 48 64 128 256; do install -Dm644 /app/jpdftweak/icons/icon${size}.png /app/share/icons/hicolor/${size}x${size}/apps/net.sourceforge.jpdftweak.jPdfTweak.png; done"
+      ],
+      "sources": [{
+        "type": "file",
+        "url": "http://prdownloads.sourceforge.net/jpdftweak/jpdftweak-1.1.zip?download",
+        "dest-filename": "jpdftweak-dist.zip",
+        "sha256": "fe48cd8a114250e131b20be67234202c24f4d94cb3b824be789653c5f6e34a47"
+      }]
+    },
+    {
+      "name": "jpdftweak-startup",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 jpdftweak /app/bin/net.sourceforge.jpdftweak.jPdfTweak",
+        "install -Dm644 net.sourceforge.jpdftweak.jPdfTweak.desktop /app/share/applications/net.sourceforge.jpdftweak.jPdfTweak.desktop",
+        "install -Dm644 net.sourceforge.jpdftweak.jPdfTweak.appdata.xml /app/share/metainfo/net.sourceforge.jpdftweak.jPdfTweak.appdata.xml"
+      ],
+      "sources": [{
+        "type": "file",
+        "path": "jpdftweak"
+      },{
+        "type": "file",
+        "path": "net.sourceforge.jpdftweak.jPdfTweak.desktop"
+      },{
+        "type": "file",
+        "path": "net.sourceforge.jpdftweak.jPdfTweak.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I changed the filename to `net.sourceforge.jpdftweak.jPdfTweak.metainfo.xml`